Automattic/woocommerce-payments
# display 0 fee value as number instead of N/A in transaction list

In transaction listing, when fee value is 0, number will be displayed instead of N/A.

Fixes https://github.com/Automattic/woocommerce-payments/issues/8286











#### Testing instructions

- Using a US merchant, assign wcpay-discount-100-off and refresh the client cache
- Perform a test purchase (any payment method)
- Check the transactions listing page and compare it to transaction details page





-------------------


**Post merge**
